renovate.json

```
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["*"],
```

```suggestion
      "matchPackagePatterns": ["*"],
```
This implicit if only excludes are provided, tough makes sense to add it for readability. 

What is the reasoning behind that you do not want to manage dependencies outside of OTEL? 🤔